cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+finnicky and took too long with a fussy newborn.
+  + Note that many _many_ people swear by flats and prefolds as the only
+  thing that can contain newborn :poop: explosions - I fully believe this _if_
+  you can get your head around how to put them on correct.
++ **All-In-Twos (AI2s):** I have nothing against the diapers themselves (they
+do dry a lot faster than AIOs because they completely come apart), but putting them
+together is Hellish. Each brand of AI2 has specific inserts that snap into them
+so you either have to commit to one brand or you have to keep hunting for the
+right inserts. Snaps are often different across diaper brands too, not to mention
+the fit of the inserts inside the shell. It makes the whole exercise both a pain
+and difficult to be picked up by anyone else.
+  + I should note that the supposed advantage of AI2s is that when you change
+  the diaper you can theoretically snap out the inserts, wipe off the shell and
+  then reuse that shell with fresh inserts, making them great for limited
+  space and holiday stashes that dry quickly. But in practice, quite a few
+  AI2 shells can't be reused as is the case with the wildly popular
+  [Close Pop-In](https://closeparent.com/collections/birth-to-potty-nappies/products/single-printed-nappy-bamboo-puffin-bio-laminate)
+  diapers that have an fleece section in the shell which is impossible to
+  clean between changes
++ **AI2s masquerading as AIOs:** There are plenty of diaper brands that have
+'AIO' diapers that are in reality, as far as I can see, AI2s. They have snap-in
+inserts that are particular to that diaper such as the Petite Crown Trima below.
+  + The advantage is that Yes, they dry faster. But this also defeats the
+  purpose of an AIO.
+
+https://petitecrown.com/collections/pocket-diapers/products/trima
+<img width="542" alt="image" src="https://user-images.githubusercontent.com/4185328/169338055-bc6d9347-cf20-40e6-9b95-64499a089a38.png">
+
+
+
+
+### Exceptions to the (my) Rules
+#### AI2s
+Although I really don't like AI2s, there is one kind that made it into our
+stash that I have not sold on and will not be selling: Close Pop-in AI2s.
+
+Let me be clear: these are possibly the _most_ annoying AI2s on the market.
+Putting them together is a total nightmare of tiny poppers that are not only
+different from every other diaper on the market but _different from one side
+of the diaper to the other_ :fire:
+I find them impressively infuriating.
+
+BUT.
+
+For _months_ (months 3 - 8 to be precise), they were the only diapers that
+I would trust to not have a major :poop: leak when we left the house.
+They are Hellish to put together, but their fit is worth gold for a few months :gem:
+
+https://closeparent.com/collections/pop-in/products/single-printed-nappy-bamboo-2021
+<img width="498" alt="image" src="https://user-images.githubusercontent.com/4185328/170536819-27882f9e-4a3b-4e55-bece-9bb49099f8f6.png">
```

Comment:
Another instance where the viewport of your device is artificially constraining the image without any benefit:
```suggestion
<img width="900" alt="image" src="https://user-images.githubusercontent.com/4185328/170536819-27882f9e-4a3b-4e55-bece-9bb49099f8f6.png">
```